news/bugfix-3863.md

```diff
@@ -0,0 +1,5 @@
+filters: fix `not` operator in filter expressions
```

Comment:
I think we should include which version was buggy.

Reply:
I've mentioned the affected version.